docker/lr-softclipper/Makefile

```diff
@@ -0,0 +1,18 @@
+VERSION = 0.5.0
+TAG1 = quay.io/broad-long-read-pipelines/lr-softclipper:$(VERSION)
+TAG2 = quay.io/broad-long-read-pipelines/lr-softclipper:latest
+TAG3 = us.gcr.io/broad-dsp-lrma/lr-softclipper:$(VERSION)
+TAG4 = us.gcr.io/broad-dsp-lrma/lr-softclipper:latest
+
+all: build push
+
+build:
+	cp ../../scripts/soft_clipper.py .
```

Comment:
Once the scripts are moved to this directory, the Makefile shouldn't need to copy and then remove any files to build the Docker image.